src/utils/index.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable camelcase */
```

Comment:
Is there a reason we are disabling eslint for this file?

Reply:
Still that old error. I obviously forgot to remove it after changing to camelCase. I'm sorry about it